back/psychoApp/models/terapist.py

```diff
@@ -66,7 +66,7 @@ def create_superuser(
 # we can confirm that the profesional_registretion is real.
```

Comment:
Also please if you can change the comment above to use therapist instead of terapist (line 5 to 7)